examples/templates/docker-devcontainer/main.tf

```
@@ -1,8 +1,7 @@
 terraform {
   required_providers {
     coder = {
-      source  = "coder/coder"
-      version = "~> 1.0.0"
+      source = "coder/coder"
```

```suggestion
      source  = "coder/coder"
      version = "~> 2.0"
```
Let's use the v2.0 of the provider.